### PR TITLE
Patch 5 for Stream Deck Name Setting

### PR DESCRIPTION
**Pull request by TheSquall**
Added two commands (!gfn, !gsn) that allow for users to input ghost names using Stream Deck.
-!gfn acts the same as !gn, and was only added to allow for continuity. This command can be removed if desired.
-!gsn takes the first word from the ghost name and adds the text afterwards.
Added ability to auto-capitalize ghost names (for those pesky times when you accidentally hit the caps lock button)

### CODE REVIEW

File: phasmophobia_evidence_v3/widget.js [hunk 1]
```diff
@@ -677,7 +724,19 @@ const resetGhost = (newName, state) => {
...
+      nameSplit = newName.split(" ");
+      for (let i = 0; i < nameSplit.length; i++) {
+        nameSplit[i] = nameSplit[i][0].toUpperCase() + nameSplit[i].substr(1).toLowerCase();
+      }
+      fullName = nameSplit.join(" ");
+    } else {
+      fullName = newName;
+    }
+  }
+  state.ghostName = fullName;
```
Comment (GlitchedMythos) [inline phasmophobia_evidence_v3/widget.js:739]: This feels repeated a lot between the three methods. Specifically the capitalization piece.

Every time we separate the word by whitespace into an array, then iterate through, uppercase, then rejoin.

I think we should move that out into a method that does the following:

```
// Takes a string and returns a string with each character of each word capitalized
const camelCase = (sentence) = > {
  return sentence.replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase());
}
```

This uses a regular expression that should capture the first letter of every word and capitalize it.

**Comment by TheSquall** [comment]
Will take a look on Wednesday!

File: phasmophobia_evidence_v3/widget.js (hunk 1 above)
Comment (TheSquall) [reply phasmophobia_evidence_v3/widget.js]: Changes were implemented in Update 26fc4a7

**Comment by TheSquall** [comment]
Incorrectly copied HTML instead of JS when committing 434de36. Fixed with 406002e. Actual changes made in 434de36 are on lines 717-721.

**Review by GlitchedMythos — APPROVED** [review]
Looks good. Cleaner than before and more easily readable

**Pull request merged**